SCAutolib/models/gui.py

```diff
@@ -0,0 +1,281 @@
+from SCAutolib import run
+
+import os
+from time import sleep, time
+import cv2
+import numpy as np
+import pandas as pd
+from io import StringIO
+import pytesseract
+import keyboard
+import uinput
+import functools
+
+
+class Screen():
+    """Captures the screenshots"""
+
+    def __init__(self, directory: str):
+        self.directory = directory
+        self.screenshot_num = 1
+
+    def screenshot(self):
+        """
```

Comment:
Please use Sphinx format of docstrings as is used in rest of this repo. Parameters and return values should be described in docstring.